app/controllers/v2/grouped_content_and_links_controller.rb

```diff
@@ -0,0 +1,26 @@
+module V2
+  #
+  # This controller provides endpoints for querying content and links together.
+  #
+  class GroupedContentAndLinksController < ApplicationController
```

Comment:
I'm not sure - I don't think either name describes this very well. Does anyone have any other suggestions?